src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -3344,19 +3325,12 @@ void ScreenBufferTests::DeleteCharsNearEndOfLine()
 
     VERIFY_ARE_EQUAL(til::point(mainView.Width() - 1, 0), mainCursor.GetPosition());
 
-    Log::Comment(NoThrowString().Format(
-        L"row_i=[%s]",
-        tbi.GetRowByOffset(0).GetText().c_str()));

```

Comment:
`GetText()` now returns a `std::wstring_view` instead of a `std::wstring`, so there's no `c_str()` function we call anymore. Changing this code to work with string_views felt very clunky (you have to pass two separate parameters to the `Format` function each time) and so I decided to remove it instead, as I felt like that if push comes to shove, you can just attach a debugger to the test case, especially now that the TAEF integration in Visual Studio works out of the box.

Reply:
also, our tests are _incredibly_ noisy. removing this is great haha

it should only really log contents on failures